libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -74,26 +74,29 @@ class AC_AttitudeControl {
     virtual AC_PID& get_rate_pitch_pid() = 0;
     virtual AC_PID& get_rate_yaw_pid() = 0;
 
-    // Gets the roll acceleration limit in centidegrees/s/s
-    float get_accel_roll_max() { return _accel_roll_max; }
+    // get the roll acceleration limit in centidegrees/s/s or radians/s/s
+    float get_accel_roll_max() const { return _accel_roll_max; }
+    float get_accel_roll_max_radss() const { return radians(_accel_roll_max*0.01f); }
```

Comment:
txs again. I'll remove the duplicate calls from the protected area.  This may have been a merge error on my part.  In any case, I suspect Leonard wanted the new functions placed up with their equivalent functions returning centi-degrees... also i suspect he wants to change the main vehicle code to use the new _radss versions in the near-ish future.. of course we could just as easily move the new functions up when that is done.